### ISSUE TITLE
Invisible health bar when a bot is one tile above the CPU

### ISSUE THREAD

**Issue by TwitchNomad** [Adam13531/BotLand#168]
**Steps to reproduce**
Move a damaged bot (health bar visible) to the tile right above the CPU.

**Expected outcome**
The health bar of the bot should be visible.

**Actual outcome**
The health bar of the bot is NOT visible.
![invisible health bar](https://user-images.githubusercontent.com/36651021/36403580-5fdb9f32-15ee-11e8-863f-694b3b676a82.jpg)

**Browser information**
{"name":"Chrome","chrome":true,"version":"63.0","blink":true,"windows":true,"a":true}

**In-game information**
Player ID: 19
Player name: Nomad

**Comment by Adam13531** [comment]
Good catch! This is a bug due to how the CPU image is currently being used in the game. It has the green tile as *part of the sprite* as opposed to being a transparent image that gets placed on a tile. This is the same reason why the tile can't be highlighted red:

![image](https://user-images.githubusercontent.com/7192897/36461997-a429e5ea-1675-11e8-9f2d-a3f70ae9210f.png)
